admin/services/data-access/pupil-census-import.data.service.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
+const { TYPES } = require('tedious')
+const moment = require('moment')
+
+const table = '[pupil]'
```

Comment:
can we please just include the table in the string.  it doesnt need to be configurable

Reply:
@GuyHarwood done